components/portal/widgets/partials/compact-widget-card.html

```diff
@@ -19,13 +19,24 @@
 
 -->
 <md-card class="list-content" id="widget-id-{{::widget.nodeId}}">
-  <md-button class="widget-action widget-info md-icon-button" aria-label="{{ widget.title }}: {{ widget.description }}" role="tooltip" ng-hide="widget.lifecycleState === 'MAINTENANCE'">
-    <md-tooltip md-direction="top" class="widget-action-tooltip">
-      {{ widget.description }}
-    </md-tooltip>
-    <md-icon>info</md-icon>
-  </md-button>
-  <div ng-transclude></div>
+  <!-- Widget contextual menu -->
+  <md-menu class="widget-action" md-position-mode="target-right bottom">
+    <md-button class="md-icon-button" aria-label="open {{ widget.title }} menu" ng-click="$mdOpenMenu($event)">
+      <md-tooltip class="widget-action-tooltip" md-direction="top" role="tooltip">
+        Open {{ widget.title }} menu
+      </md-tooltip>
+      <md-icon>more_vert</md-icon>
```

Comment:
While this menu opener button *is* in the tab order when keyboard navigating, the icon does not visually appear when this button is selected via keyboard tab. It'd presumably be more usable if the button did appear when selected, so that users navigating a typical browser via keyboard have a stronger visual cue as to what they've selected. (The tooltip does appear, so it's kind of a tooltip-on-an-invisible-menu experience.)

![image 2018-07-17 at 7 15 04 am](https://user-images.githubusercontent.com/952283/42816704-4dcd41f8-8991-11e8-891e-add2336f5e04.png)

Maybe the feature would be more discoverable and usable in general if the menu actuator were to always show, rather than appearing on hover (or selection).

This changeset doesn't introduce this issue (it was already this way for expanded mode widgets), just continues it.

Reply:
>Maybe the feature would be more discoverable and usable in general if the menu actuator were to always show, rather than appearing on hover (or selection).

Now that there's slightly less "stuff" in the heading space, I like this change. Added it with screenshot example.